R/functions.R

```diff
@@ -3,6 +3,44 @@
 #' Return id of the planning googlesheet
 gs_id <- function() "1HLtyGK_csi5W_v7XChxgTuVjS-RKXqc0Jxos1RBqpwk"
 
+
+#' Sanity checks for the google sheet.
+#' This function is used in data pipe-lines to check that everything
+#' in the google sheet is as expected.
+#' It will throw errors or warnings if something goes wrong.
+#'
+#' @details the function can be used in a dplyr pipe, which is why
```

Comment:
So when you distinguished a description paragraph (which is recommended), the `@details` tag becomes optional. But it's OK of course; also for description one can use `@description` explicitly.

In absence of a description, a description is always generated by roxygen: a copy of the title.

Purely FYI.